package.json

```diff
@@ -43,7 +43,7 @@
 		"@godaddy/terminus": "^4.11.0",
 		"@graphql-tools/load": "^7.7.0",
 		"@graphql-tools/url-loader": "^7.17.18",
-		"@kiva/kv-components": "^6.59.4",
+		"@kiva/kv-components": "^6.62.0",
```

Comment:
@tiarascoleman looks like your lock file jumped versions. Which version of npm and node are you using locally? We'll need to revert the lock file and regenerate with the existing version.

Reply:
I'm on this version: node v20.19.5 (npm v10.8.2)

Reply:
Sent possible fix over slack to prevent lock version changes.